osm2streets/src/paint.rs

```diff
@@ -0,0 +1,235 @@
+use crate::{marking, marking::Marking};
+
+// We use geom and stay in map space. Output is done in latlon.
+use geom::{Angle, Distance, Line, PolyLine, Polygon, Pt2D, Ring};
+
+use crate::lanes::TrafficMode;
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct PaintArea {
+    // Because I'm lazy and don't want to make different "map space" and "lonlat space" PaintArea
```

Comment:
I've thought about making some of the `geom` types take a generic parameter to distinguish this more carefully, but it's felt like overkill so far. (And also all of the math would have to change!)

Reply:
A fully generic geom library with all the algorithms would be hectic!